hack/utils.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -x
+
+function running_from_skipper() {
+   # The SKIPPER_UID environment variable is an indication that we are running on a skipper container.
+   [ -n "${SKIPPER_UID+x}" ]
+}
+
+function get_container_runtime_command() {
+    if running_from_skipper; then
+        # skipper creates the SKIPPER_DOCKER_GID env variable for a docker based build
+        CONTAINER_TOOL=$( [[ "$CONTAINER_RUNTIME_COMMAND" == ""docker"" ]] && echo "docker" || echo "podman-remote")
```

Comment:
Did you intend to check the `SKIPPER_DOCKER_GID`? Otherwise, I don't understand this comment.

Reply:
sorry:) old junk